Makefile

```diff
@@ -37,6 +37,17 @@ test: validate-manifest lint unit-test cleanup-integration-test integration-test
 unit-test:
 	pytest assisted-events-scrape/tests/unit
 
+ci-integration-test:
+	./tools/generate_mockserver_configmap.sh | kubectl delete -f - || true
+	./tools/generate_mockserver_configmap.sh | kubectl create -f - || true
+	oc apply -f assisted-events-scrape/tests/integration/manifests
+	oc wait --timeout=120s --for=condition=Available deployment --all
+	oc wait --timeout=300s --for=condition=Ready pods --all
+	oc apply -f openshift/
+	oc wait --timeout=30s --for=condition=Available deployment --all
+	oc wait --timeout=30s --for=condition=Ready pods --all
+	ES_SERVER=elasticsearch-master:9200 ES_INDEX=assisted-service-events pytest assisted-events-scrape/tests/integration
```

Comment:
I find it a bit problematic to duplicate that much

Just a proposal: can we put it in a bash script that has different behavior for the parts that are different?
It will also leave the makefile a bit shorter

Reply:
Good idea